lib/fastly/big_query_logging.rb

```diff
@@ -0,0 +1,90 @@
+class Fastly
+  # An Google BigQuery endpoint to stream logs to
+  class BigQueryLogging < BelongsToServiceAndVersion
+    attr_accessor :created_at, :dataset, :format, :format_version, :name, :placement, :project_id,
```

Comment:
I think you can remove `service_id` and `version` from this since we're inheriting from `BelongsToServiceAndVersion`

Reply:
I copied this from `GcsLogging`.

`BelongsToServiceAndVersion` has `attr_writer :version` and both its `#service` and `#version` methods depend on `#service_id`, but there's no definition of `#service_id`.

But I'm still confused about `attr_accessor :version`. The parent has `attr_writer :version` and a custom `#version` method, so this would redefine the writer with the same thing and redefine the reader with something else. I don't know why.

Reply:
Turns out that we didn't need the `attr_accessor :version` but it also wasn't hurting anything.